src/index.css

```diff
@@ -12,10 +12,10 @@ code {
     monospace;
 }
 
-/* #btn {
+#btn {
   border-radius: 8px;
   height: 35px;
   width: 114px;
   font-size: 18px;
   background-color: #ffffff;
```

Comment:
is this block being used?

Reply:
also color properties that are more that are the same character greater than 3 can be converted to shorthand like:

```css
background-color: #fff;
```
That's a general convention that is used.

Reply:
It's to style buttons.